app/views/shared/_google_analytics_snippet.slim

```diff
@@ -0,0 +1,2 @@
+script
+  |  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){ (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o), m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m) })(window,document,'script','//www.google-analytics.com/analytics.js','ga'); ga('create', 'UA-26370633-3', 'auto'); ga('send', 'pageview'); 
```

Comment:
@paulaferris this looks good. It would be perfect if the ga code wasn't fixed in the codebase. I'd suggesting storing the ID in `Settings` - see https://github.com/SumOfUs/Champaign/blob/development/app/views/layouts/liquid.slim#L6 for an example of `Settings` in use.